tests/vmi_multus_test.go

```diff
@@ -356,11 +362,11 @@ var _ = Describe("Multus", func() {
 				tests.WaitUntilVMIReady(vmiOne, tests.LoggedInAlpineExpecter)
 				tests.WaitUntilVMIReady(vmiTwo, tests.LoggedInAlpineExpecter)
 
-				configInterface(vmiOne, "eth1", "10.1.1.1/24", "localhost:~#")
+				configInterface(vmiOne, "eth1", CIDR1, "localhost:~#")
 				By("checking virtual machine interface eth1 state")
 				checkInterface(vmiOne, "eth1", "localhost:~#")
 
-				configInterface(vmiTwo, "eth1", "10.1.1.2/24", "localhost:~#")
+				configInterface(vmiTwo, "eth1", CIDR2, "localhost:~#")
 				By("checking virtual machine interface eth1 state")
 				checkInterface(vmiTwo, "eth1", "localhost:~#")
 
```

Comment:
ditto for the following `By`

Reply:
https://github.com/kubevirt/kubevirt/pull/3479/commits/134316abe3871790e763e2cd8e97a7f8bf31e349#r468607003